internal/attest/attest.go

```diff
@@ -226,18 +228,32 @@ func encode(store storage.Storer, enc Encoder) (plumbing.Hash, error) {
 	return store.SetEncodedObject(obj)
 }
 
+func generateStatement(pred []byte, attType string, sha plumbing.Hash) (*spb.Statement, error) {
+	sub := []*spb.ResourceDescriptor{{
+		Digest: map[string]string{"gitCommit": sha.String()},
```

Comment:
This we're probably going to need to plumb through some additional data to determine if this should be gitCommit, gitTree, etc. but i'm okay with starting with this.

Reply:
Right, that will also require changing how that templating works.  I'm somewhat wondering if it would be easier to drop the file based templating and instead construct the JSON 'inline' in code.  WDYT?

Reply:
This is for the test output? yeah that's fine with me!

Reply:
Let me fix that now.